### ISSUE TITLE
Differences ISO122-2 ISO122-3

### ISSUE THREAD

**Issue by fabiopuddu77** [usnistgov/AGA8#22]
Hello @EricLemmon ,

Is the ISO122-2 your Detail formula and the ISO122-3 the GROSS formula in your code?

**Comment by EricLemmon** [comment]
That's correct, although the reference states for enthalpy and entropy might be different.

**Comment by fabiopuddu77** [comment]
If I change AGA should I change heating values at 25 as well? Because the ISO122-3 is the AGA8 but in your code there is heating values from AGA 5.
